pkg/cmd/argtoenv.go

```diff
@@ -12,6 +12,7 @@ func AddEnvToUsage(cmd *cobra.Command, argToEnv map[string]string) {
 	flagSet := make(map[string]bool)
 
 	for arg, env := range argToEnv {
+		//nolint:errcheck
```

Comment:
Not sure if we want to litter our code with linter instructions.  We could ignore those in the config https://github.com/cloudfoundry-incubator/quarks-operator/commit/29e571ffbc464dec2c62cc9a259b97169ae0df65

What do you think?